fe/fe-core/src/main/java/org/apache/doris/load/Load.java

```diff
@@ -1610,7 +1612,8 @@ public boolean isLabelExist(String dbName, String labelValue, boolean isAccurate
                 }
             } else {
                 for (Map.Entry<String, List<LoadJob>> entry : labelToLoadJobs.entrySet()) {
-                    if (entry.getKey().contains(labelValue)) {
+                    PatternMatcher matcher = PatternMatcher.createMysqlPattern(labelValue, CaseSensibility.LABEL.getCaseSensibility());
```

Comment:
`PatternMatcher.createMysqlPattern` can be put outside the `for` loop.
Same suggestion to other `PatternMatcher.createMysqlPattern`.

Reply:
+1